### PR TITLE
fix pointsGroups web

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

File: OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java [hunk 1]
```diff
@@ -507,6 +511,28 @@ private void updatePoint(double lat, double lon, String description, String name
 				setBackgroundType(backgroundType);
 			}
 		}
+		
+		private void updatePoint(double lat, double lon, String description, String name,
```
Comment (vshcherb) [inline OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java:515]: duplicate method

File: OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java [hunk 2]
```diff
@@ -1569,6 +1595,25 @@ public boolean deleteWptPt(WptPt point) {
 
 			return points.remove(point);
 		}
+		
+		public boolean deleteWptPt(String wptName) {
```
Comment (vshcherb) [inline OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java:1599]: class is overloaded with lines of code, refactoring is needed and this method is not a good example that should be here especially wptName is not unique key!

File: OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java [hunk 3]
```diff
@@ -1599,6 +1663,25 @@ public void updateWptPt(WptPt point, double lat, double lon, String description,
 			modifiedTime = System.currentTimeMillis();
 			pointsModifiedTime = modifiedTime;
 		}
+		
+		public void updateWptPt(WptPt point, double lat, double lon, String description, String name,
+		                        String category, String color, String iconName, String backgroundType, String address) {
```
Comment (vshcherb) [inline OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java:1668]: class is overloaded with lines of code, refactoring is needed and this method is not a good example that should be here especially wptName is not unique key!

File: OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java (hunk 1 above)
Comment (alisa911) [reply OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java:515]: this is overload method from android. i can't change safely such methods

File: OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java (hunk 2 above)
Comment (alisa911) [reply OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java:1599]: wptName is unique key for group

**Pull request merged**